src/tools/cephfs/cephfs-shell

```diff
@@ -360,6 +366,14 @@ class CephFSShell(Cmd):
     def default(self, line):
         perror('Unrecognized command')
 
+    def set_retval_from_except(self, ex):
+        """
+        Sets the exit code/return value for cephfs-shell.
+
+        :param ex: exception
+        """
+        self.exit_code = ex.errno if hasattr(ex, 'errno') else 1
```

Comment:
Why are there two of these methods?

Also, let's add a method to `libcephfs.Error`:

````
def get_error_code(self):
  return 1
````

Then overload in `libcephfs.OSError` to return `self.errno`.

Reply:
> Why are there two of these methods?

I'm sorry. It's a mistake. It's part of some of an issue I was facing during git rebase yesterday. 

> Also, let's add a method to `libcephfs.Error`:
> 
> ```
> def get_error_code(self):
>   return 1
> ```
> 
> Then overload in `libcephfs.OSError` to return `self.errno`.

Sounds great.

Reply:
I've made the modification as requested-